lerobot/scripts/train.py

```diff
@@ -349,8 +349,7 @@ def evaluate_and_checkpoint_if_needed(step):
                 start_seed=cfg.seed,
             )
             log_eval_info(logger, eval_info["aggregated"], step, cfg, offline_dataset, is_offline)
-            if cfg.wandb.enable:
-                logger.log_video(eval_info["video_paths"][0], step, mode="eval")
+            logger.log_video(eval_info["video_paths"][0], step, mode="eval")
```

Comment:
Why not add a `cfg.tensorboard.enable` argument to support both?

Reply:
Good idea! Will do.